content/cloud-docs/agents/hooks.mdx

```diff
@@ -0,0 +1,137 @@
+---
+page_title: Hooks - Terraform Cloud Agents - Terraform Cloud and Terraform Enterprise
+---
+
+# Hooks
+
+Terraform Cloud Agents support executing custom programs, or hooks, during
+strategic points of a Terraform run. These hooks allow operators to extend the
+functionality of Terraform runs.
+
+Terraform Cloud Agents support the following hooks:
```

Comment:
Could we add a heading here? I wonder if it's Hook Types or Hook Run Stages, or something - thoughts?  Also question - I know that we can't choose a custom name. Can we only have one hook of each type within an agent?

Reply:
I added a heading. Yep, only one hook per type.